Lab 1/README.md

```diff
@@ -45,11 +55,11 @@ To push an image:
 
 8. Now push that image up to IBM Cloud Container Registry: 
 
-   ```docker push registry.ng.bluemix.net/<namespace>/hello-world```
+   ```docker push registry.ng.bluemix.net/<my_namespace>/hello-world```
```

Comment:
We never explain the angle bracket syntax, `<>` which I'm assuming is "the person who is going through the steps own choice of content". 

`namespace` did not match `my_namespace` above and I think it should.